zypp/target/RpmPostTransCollector.cc

```diff
@@ -88,10 +90,36 @@ namespace zypp
 
 	  Pathname noRootScriptDir( ZConfig::instance().update_scriptsPath() / tmpDir().basename() );
 
-	  for ( const auto & script : _scripts )
+	  ProgressData scriptProgress( static_cast<long long>(_scripts.size()) );
+	  callback::SendReport<ProgressReport> report;
+	  scriptProgress.sendTo( ProgressReportAdaptor( ProgressData::ReceiverFnc(), report ) );
+
+	  bool firstScript = true;
+	  for ( const auto &script : _scripts )
 	  {
+	    const std::string & pkgident( script.substr( 0, script.size()-6 ) );	// strip tmp file suffix
+
+	    scriptProgress.name( str::Format(_("Executing %posttrans script '%1%'")) % pkgident );
```

Comment:
Did you test it? Looks scrambled for me:
> Executing allscriptA-1-0.intern.noarch.rpmosttrans script 'allscriptA-1-0.intern.noarch.rpm'

Reply:
Done